### PR TITLE
FIX- Dropdown in Navbar Not Visible on Hover #308

### LINKED ISSUE #308: Dropdown in Navbar Not Visible on Hover
#### Current Behavior
On the [docs.layer5.io](https://docs.layer5.io/) website, the dropdown in the navbar for "Products" isn't visible when hovered over; it only works on click.

#### Expected Behavior
The dropdown menu should be visible when the "Products" option in the navbar is hovered over.

#### Screenshots/Logs
Screenshot (12)

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by 2003Aditya**
**Notes for Reviewers**

This PR fixes #308 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ashparshp** [comment]
LGTM!!

**Review by Ashparshp — COMMENTED** [review]
For hover it's fine!! but for phone view it's having issues, note: it must be like as it was before..

File: assets/scss/_navbar_project.scss [hunk 1]
```diff
@@ -9,10 +9,14 @@
   .current {
     color: $primary !important;
   }
-
+  &:hover {
```
Comment (Ashparshp) [inline assets/scss/_navbar_project.scss:12]:
```suggestion
    @media (min-width: 768px) {
    &:hover {
      .dropdown-menu {
        display: block;
      }

      .dropdown-menu {
        transform: translate(-15%, 0rem) !important;
    }

  }
  }
```

File: assets/scss/_navbar_project.scss [hunk 2]
```diff
@@ -9,10 +9,14 @@
...
+    .dropdown-menu {
+      display: block;
+    }
+  }
   .dropdown-menu {
     padding: 0;
-    transform: translate(-15%, 2.5rem) !important;
+    transform: translate(-15%, 0rem) !important;
```
Comment (Ashparshp) [inline assets/scss/_navbar_project.scss:19]:
```suggestion
    transform: translate(-15%, 2.5rem) !important;
```

**Review by Ashparshp — CHANGES_REQUESTED** [review]

File: assets/scss/_navbar_project.scss (hunk 2 above)
Comment (Ashparshp) [reply assets/scss/_navbar_project.scss]: @leecalcote I noticed that the !important wasn't originally necessary, and upon further testing, the dropdown behavior is functioning correctly without it. It seems someone might have added it mistakenly. Hence, we can proceed without using !important at all.

Comment (2003Aditya) [reply assets/scss/_navbar_project.scss]: I will those important then

**Review by Ashparshp — APPROVED** [review]
LGTM!

**Comment by vishalvivekm** [comment]
@2003Aditya 
Thank you for your contribution.
Let's discuss this on Websites' call starting in 30 minutes. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by 2003Aditya** [comment]
@vishalvivekm I'm sorry I couldn't join at that time..

**Comment by leecalcote** [comment]
@iArchitSharma able to review?

**Review by hargunkaur286 — APPROVED** [review]

**Comment by hargunkaur286** [comment]
Looks good to me!

**Comment by hargunkaur286** [comment]
@2003Aditya Thank you for your contribution. Let's discuss it in today's website meeting. Adding this as an agenda item in [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit)

**Comment by Ashparshp** [comment]
@2003Aditya Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

Any luck?

**Comment by Ashparshp** [comment]
@hargunkaur286, ready to merge?

**Pull request merged**